demos-europe/demosplan-core
# chore: Upgrade ancient geo library

phayes/geophp was not updated for a long time and did not even use namespaces. This lead to some hacks that could be avoided with the usage of a fork where namespaces are introduced.

### How to review/test
create a statement with a  drawing and export it as docx. You may see a picture of it

### Linked PRs (optional)


### Tasks (optional)


### PR Checklist


Delete the checkbox if it doesn't apply/isn't necessary.
